app/controllers/hyrax/batch_edits_controller.rb

```diff
@@ -39,7 +39,15 @@ def destroy_collection
         obj = ActiveFedora::Base.find(doc_id, cast: true)
         obj.destroy
       end
-      flash[:notice] = "Batch delete complete"
+
+      admin_set_name = "Administrative Set"
+      admin_set_name = AdminSet.first.title.first unless AdminSet.first.nil?
+
+      flash[:notice] = if flash[:notice].nil?
+                         "Batch delete complete"
+                       else
+                         I18n.t('errors.messages.batch_delete_collections_error', admin_set_name: admin_set_name)
```

Comment:
Won't this show up even when the delete is successful?  What about when the batch is empty? Will it overwrite the 'Select something first' message?